Lib/logging/__init__.py

```diff
@@ -2052,11 +2045,7 @@ def critical(msg, *args, **kwargs):
 
 def fatal(msg, *args, **kwargs):
     """
-    Log a message with severity 'CRITICAL' on the root logger. If the logger
-    has no handlers, call basicConfig() to add a console handler with a
-    pre-defined format.
-
-    The critical() function is preferred.
+    Don't use this method, use critical() instead.
```

Comment:
Should say "function" here rather than "method".

Reply:
Yes, I fixed it.